### ISSUE TITLE
OS X - sudo requirement for ./letsencrypt-auto --help --debug

### ISSUE THREAD

**Issue by rubenvarela** [certbot/certbot#1521]
Why is sudo required when I run ./letsencrypt-auto --help --debug (OS X)?
1. As far as I can tell, all the dependencies are installed before this step. 
2. The dependencies are installed using Homebrew, so everything is owned by the local user.
3. **I only need want the --help output.**

This is the last line before asking for the password:

```
Running with virtualenv: sudo /Users/ruben/.local/share/letsencrypt/bin/letsencrypt --help --debug
```

The full output I get when running the command is:

```
./letsencrypt-auto --help --debug
grep: /etc/os-release: No such file or directory
Bootstrapping dependencies for Mac OS X...
==> Installing dependencies for augeas: libxml2
==> Installing augeas dependency: libxml2
==> Downloading https://homebrew.bintray.com/bottles/libxml2-2.9.2.el_capitan.bottle.1.tar.gz
######################################################################## 100.0%
==> Pouring libxml2-2.9.2.el_capitan.bottle.1.tar.gz
==> Caveats
This formula is keg-only, which means it was not symlinked into /usr/local.

OS X already provides this software and installing another version in
parallel can cause all kinds of trouble.

Generally there are no consequences of this for you. If you build your
own software and it requires this formula, you'll need to add to your
build variables:

    LDFLAGS:  -L/usr/local/opt/libxml2/lib
    CPPFLAGS: -I/usr/local/opt/libxml2/include

==> Summary
🍺  /usr/local/Cellar/libxml2/2.9.2: 275 files, 10M
==> Installing augeas
==> Downloading https://homebrew.bintray.com/bottles/augeas-1.4.0.el_capitan.bottle.1.tar.gz
######################################################################## 100.0%
==> Pouring augeas-1.4.0.el_capitan.bottle.1.tar.gz
==> Caveats
Lenses have been installed to:
  /usr/local/share/augeas/lenses/dist
==> Summary
🍺  /usr/local/Cellar/augeas/1.4.0: 420 files, 3.9M
==> Downloading https://homebrew.bintray.com/bottles/dialog-1.2-20150528.el_capitan.bottle.tar.gz
######################################################################## 100.0%
==> Pouring dialog-1.2-20150528.el_capitan.bottle.tar.gz
🍺  /usr/local/Cellar/dialog/1.2-20150528: 13 files, 824K
Creating virtual environment...
Updating letsencrypt and virtual environment dependencies.......
Running with virtualenv: sudo /Users/ruben/.local/share/letsencrypt/bin/letsencrypt --help --debug
Password:    
```

**Comment by smarek** [comment]
This is not an issue just for OSX, on my Debian box it requires sudo authorization before showing help too. I'd be more glad to have the cert deployment process under control, by knowing why I should provide sudo access (and in my case sudo is not even enabled)

**Comment by bmw** [comment]
This was fixed in version 0.6.0.
